src/Sulu/Component/Security/Authorization/SecurityCondition.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Security\Authorization;
+
+/**
+ * The class which describes the necessary permissions to access a certain element
+ * @package Sulu\Bundle\SecurityBundle\Security
+ */
+class SecurityCondition
+{
+    /**
+     * The string representation of the security context
```

Comment:
These descriptions are duplicated three times in this class. I would only document the public API